core/java/iesi-core/pom.xml

```diff
@@ -293,6 +293,12 @@
             <version>3.16.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jetbrains</groupId>
+            <artifactId>annotations</artifactId>
+            <version>RELEASE</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
Why is this dependency added?

Reply:
I should have deleted it, I used it to add specific collections on TextSubstring class when I thought I should catch only the **first word** on the function substring